### ISSUE TITLE
Preview images are not responsive

### ISSUE THREAD

**Issue by klihelp** [Sommerregen/grav-plugin-mediaembed#28]
Preview images loads only one size for all screens..

**Comment by Sommerregen** [comment]
@klihelp I totally agree, but unfortunately there is nothing we can change. The oembed protocol doesn't support/provide multiple preview images for different sizes/resolutions of the screen :disappointed: and there is no general way to find preview images for all services...

**Comment by metbril** [comment]
You could try to use Grav image manipulation to render preview images?

https://learn.getgrav.org/content/media#image-actions
